ext/nnstreamer/tensor_filter/tensor_filter_python3.hh

```diff
@@ -0,0 +1,131 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2024 Yelin Jeong <yelini.jeong@samsung.com>
+ */
+/**
+ * @file    tensor_filter_subplugin_python3.hh
+ * @date    26 Mar 2024
+ * @brief   NNStreamer tensor-filter subplugin python header
+ * @see     https://github.com/nnstreamer/nnstreamer
+ * @author  Yelin Jeong <yelini.jeong@samsung.com>
+ * @bug     No known bugs
+ */
+
+#ifndef __TENSOR_FILTER_SUBPLUGIN_PYTHON3_H__
+#define __TENSOR_FILTER_SUBPLUGIN_PYTHON3_H__
+
+/* nnstreamer plugin api headers */
+#include <map>
+#include <nnstreamer_conf.h>
+#include <nnstreamer_cppplugin_api_filter.hh>
+#include <nnstreamer_log.h>
+#include <nnstreamer_util.h>
+#include "nnstreamer_python3_helper.h"
+
+/** @brief Callback type for custom filter */
+typedef enum _cb_type
+{
+  CB_SETDIM = 0,
+  CB_GETDIM,
+
+  CB_END,
```

Comment:
If you make your code use these enum in the class, you can define it inside the class. Or, keep it in the namespace (encapsulation)

Reply:
Used namespace!
Thank you!